cves/kernel/CVE-2017-10911.yml

```diff
@@ -107,14 +105,6 @@ vcc_instructions: |
 vccs:
 - commit: 4d05a28db56225bbab5e1321d818f318e92a4657
   note: Discovered automatically by archeogit.
-- commit: 8812293323a79134e06c3bf82eba1e217d23382e
```

Comment:
These all appear to be contributing to the vulnerability. What was the reason for removing them?

Reply:
I moved those commits to "interesting commits". The initial commit already pushed vulnerable code, and the following commits were just adding stuff to the code that was already unsafe.